estate/views/estate_property_views.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="estate_property_view_tree" model="ir.ui.view">
+        <field name="name">estate.property.name</field>
```

Comment:
This field the the name of the view, so `estate.property.list` is probably more correct.